integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOCILoadBalancer.java

```diff
@@ -0,0 +1,295 @@
+            lbIp,
+            httpport);
+    assertThat(callWebAppAndCheckForServerNameInResponse(curlCmd, managedServerNames, 50))
+        .as("Verify OCI LB can access the sample-war app "
+            + "from all managed servers in the domain via http")
+        .withFailMessage("OCI LB can not access the the sample-war app "
+            + "from one or more of the managed servers via http")
+        .isTrue();
+  }
+
+  private static void createAndVerifyDomain(String miiImage, String domainNamespace, String domainUid) {
+    // create secret for admin credentials
+    logger.info("Create secret for admin credentials");
+    String adminSecretName = "weblogic-credentials";
+    String adminServerPodName = domainUid + "-" + ADMIN_SERVER_NAME_BASE;
+    String managedServerPrefix = domainUid + "-" + MANAGED_SERVER_NAME_BASE;
+    assertDoesNotThrow(() -> createSecretWithUsernamePassword(adminSecretName, domainNamespace,
+        ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT),
+        String.format("create secret for admin credentials failed for %s", adminSecretName));
+
+    // create encryption secret
+    logger.info("Create encryption secret");
+    String encryptionSecretName = "encryptionsecret";
+    assertDoesNotThrow(() -> createSecretWithUsernamePassword(encryptionSecretName, domainNamespace,
+        "weblogicenc", "weblogicenc"),
+        String.format("create encryption secret failed for %s", encryptionSecretName));
+
+    // create domain and verify
+    logger.info("Create model in image domain {0} in namespace {1} using docker image {2}",
+        domainUid, domainNamespace, miiImage);
+    createDomainCrAndVerify(adminSecretName, OCIR_SECRET_NAME,
+        encryptionSecretName, miiImage, domainNamespace, domainUid);
+
+    // check that admin server pod exists in the domain namespace
+    logger.info("Checking that admin server pod {0} exists in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkPodExists(adminServerPodName, domainUid, domainNamespace);
+
+    // check that admin server pod is ready and admin service exists in the domain namespace
+    logger.info("Checking that admin server pod {0} is ready in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkPodReadyAndServiceExists(adminServerPodName, domainUid, domainNamespace);
+
+    // check for managed server pods existence in the domain namespace
+    for (int i = 1; i <= replicaCount; i++) {
+      String managedServerPodName = managedServerPrefix + i;
+
+      // check that the managed server pod is ready and the service exists in the domain namespace
+      logger.info("Checking that managed server pod {0} is ready in namespace {1}",
+          managedServerPodName, domainNamespace);
+      checkPodReadyAndServiceExists(managedServerPodName, domainUid, domainNamespace);
+    }
+  }
+
+  private static void createDomainCrAndVerify(String adminSecretName,
```

Comment:
Can u explore the CommonMiiTestUtils.createMiiDomainAndVerify() method instead of creating a private method createDomainCrAndVerify()

Reply:
replaced